Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+
+verify_ssl = true
+url = "https://pypi.python.org/simple"
+name = "pypi"
+
+[packages]
+
+typing = "==3.6.4"
+urwid = "==2.0.1"
+zulip = "==0.4.2"
+
+[dev-packages]
+
+pytest = "==3.4.2"
+"pytest-pep8" = "==1.0.6"
+pytest-mock = "==1.7.1"
+pytest-cov = "==2.5.1"
+pudb = "==2017.1.4"
+snakeviz = "==0.4.2"
+mypy = "==0.560"
+
+[requires]
+
+python_version = "3.5"
```

Comment:
I'll need to look up what this `python_version` means.  Do we not support Python 3.4 or 3.6? If so, why are the configured [here](https://github.com/zulip/zulip-terminal/pull/63/files#diff-354f30a63fb0907d4ad57269548329e3R4)?

Reply:
It recommend in the pipenv docs https://docs.pipenv.org/basics/#general-recommendations-version-control

Reply:
This would mean that [other Python versions won't work](https://github.com/pypa/pipenv/issues/1050) (including `3.4` and `3.6`)

You could [remove](https://docs.pipenv.org/basics/#specifying-versions-of-python) the `python_version` requirement, but then I guess users will have to explicitly specify that they want to use Python3 (and not 2).